Tuist/ProjectDescriptionHelpers/BuildConfiguration.swift

```diff
@@ -30,13 +30,18 @@ public enum BuildConfiguration: CaseIterable {
         }
     }
 
-    public func createConfiguration(projectName: String) -> Configuration {
+    public func createConfiguration(projectName: String, compileTimeWarning: Int?) -> Configuration {
+        var settings: [String: SettingValue] = [:]
+				if let compileTimeWarning = compileTimeWarning {
+						settings["OTHER_SWIFT_FLAGS"] = "-Xfrontend -warn-long-expression-type-checking=\(compileTimeWarning) -Xfrontend -warn-long-function-bodies=\(compileTimeWarning)";
+				}
```

Comment:
Please check the indent here.
I would prefer you set up: `Prefer Indent Using: Spaces` instead of Tabs :D